lib/detect-testing-library-utils.ts

```diff
@@ -45,6 +49,9 @@ export function detectTestingLibraryUtils<
 
     // Init options based on shared ESLint settings
     const customModule = context.settings['testing-library/module'];
+    const fileNamePattern =
+      context.settings['testing-library/file-name'] ??
```

Comment:
should the setting be `'testing-library/file-name-pattern'`? Or adding the word pattern somewhere 😄  so the intention is clearer

Reply:
You are absolutely right. I'll rename it in #247 which is the next PR where I already renamed this from `file-name` to `filename`.